src/linagora.esn.calendar/i18n/vi.json

```diff
@@ -51,8 +51,8 @@
   "An user booked the resource {{name}}": "Một người dùng đã đăng kí tài nguyên {{name}}",
   "As resource manager, you must accept or decline the request": "Là người quản lí tài nguyên, bạn phải đồng ý hoặc từ chối yêu cầu",
   "Attendee availability is unknown": "Attendee availability is unknown",
-  "Attendee is available": "Người tham dự có sẵn",
-  "Attendee is not available": "Người tham dự không có mặt",
```

Comment:
I bet this is an old version of Google translator :trollface:

Reply:
:trollface: